sync/sync.go

```diff
@@ -409,13 +400,19 @@ func (sync *synchronizer) peerIsInTheCommittee(id peer.ID) bool {
 		return false
 	}
 
-	return sync.state.IsInCommittee(p.Address())
+	for key := range p.ConsensusKeys {
```

Comment:
Consensus also can tell us if we have an active validator

Reply:
No, this is about other peers in the network.